src/app/components/observation/observation/observation.component.html

```diff
@@ -110,17 +110,23 @@ <h2>
   }
 </div>
 
-<!-- Knapperad med deleknapp og endre-knapp -->
+<!-- Knapperad med deleknapp, gå til obs og endre-knapp -->
 <div class="buttons ion-color-light">
   @if (userCanEdit()) {
     <ion-button fill="clear" class="edit-button" (click)="edit()" [disabled]="isLoadingObsForEdit()">
-      <svg-icon class="edit-icon" src="/assets/icon/pencil.svg"></svg-icon>
+      <ion-icon name="create-outline"></ion-icon>
       {{ "REGISTRATION.EDIT" | translate }}
       @if (isLoadingObsForEdit()) {
         <ion-spinner name="dots" slot="end"></ion-spinner>
       }
     </ion-button>
   }
+  @if (!isRegistrationPage()) {
+    <ion-button fill="clear" class="edit-button" [href]="'registration/' + registration().RegId">
```

Comment:
For intern routing bør du bruke routerLink, ikke href, for å unngå at applikasjonen starter på nytt

Reply:
stemmer. jeg ser den er brukt på alle knapper her